packages/playwright-crawler/src/internals/adaptive-playwright-crawler.ts

```diff
@@ -264,6 +280,7 @@ export class AdaptivePlaywrightCrawler extends PlaywrightCrawler {
 
             if (plainHTTPRun.ok && this.resultChecker(plainHTTPRun.result)) {
                 crawlingContext.log.debug(`HTTP-only request handler succeeded for ${crawlingContext.request.url}`);
+                plainHTTPRun.logs?.forEach(([log, method, ...args]) => log[method](...(args as [any, any])));
```

Comment:
- This way, the log messages will be printed with +/- the same timestamp and all at once, which might confuse somebody. Are we OK with this?
- Also, since logs in browser runs are not intercepted, this effect will only manifest for http runs
- Is it OK that there will still be two log "batches" when we do a detection (run both browser and http)?

Reply:
I wouldnt care about this personally, the log abstraction is unfortunately not very friendly for doing it better, we would first need to improve that I guess. Better this than logging twice.

Reply:
Re the last point, I dont think its true, but maybe I missed something. The logs are printed only if http version is committed, we never do that if we run the browser, right?

Reply:
Right, the "capture" is done in the  runRequestHandlerWithPlainHttp function. Then it's fine.